proj will fail pkgconf test pipeline

If the pkgconf test pipeline is added to proj it will fail in the following way:

```
2024/11/12 23:14:12 INFO running step "test/pkgconf"
2024/11/12 23:14:12 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:14:12 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:14:12 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:14:12 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:14:12 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + basename /home/build/melange-out/proj-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + dev_pkg=proj-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + apk info -L proj-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep -q ^Name: usr/lib/pkgconfig/proj.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + basename usr/lib/pkgconfig/proj.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + lib_name=proj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + echo usr/lib/pkgconfig/proj.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep -q '^usr/lib/pkgconfig/proj.pc$\|^usr/share/pkgconfig/proj.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + pkgconf --exists proj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep -q ^Version: usr/lib/pkgconfig/proj.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + pkgconf --modversion proj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 INFO 9.5.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep -q ^Libs: usr/lib/pkgconfig/proj.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + pkgconf --libs proj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 INFO -lproj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + grep -q ^Cflags: usr/lib/pkgconfig/proj.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN + pkgconf --cflags proj uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN Package sqlite3 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN Perhaps you should add the directory containing `sqlite3.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN Package 'sqlite3', required by 'proj', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN Package 'libtiff-4', required by 'proj', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 WARN Package 'libcurl', required by 'proj', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:12 INFO pod bfbaed5474e0bfcfcec771605c0f028eafa092ea1a9e6e89bd2c99c8c4eb0989 terminated
2024/11/12 23:14:13 INFO pod 6b7c8ba6ffd767c091bce0961f2d88fdc0912cf6b36a62a5a3989a69696a5939 terminated
2024/11/12 23:14:13 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:14:13 INFO   workspace dir: /home/user/tmp/melange-workspace-1826686221
2024/11/12 23:14:13 INFO   guest dir: /home/user/tmp/melange-guest-1810358410
2024/11/12 23:14:13 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/proj] Error 1
```

--- a/proj.yaml
+++ b/proj.yaml
@@ -63,7 +63,12 @@ subpackages:
     dependencies:
       runtime:
         - proj
+        - curl-dev
+        - tiff-dev
     description: proj dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

--- a/proj.yaml
+++ b/proj.yaml
@@ -2,7 +2,7 @@
 package:
   name: proj
   version: 9.5.0
-  epoch: 1
+  epoch: 2
   description: PROJ coordinate transformation software library
   copyright:
     - license: MIT
@@ -63,8 +63,6 @@ subpackages:
     dependencies:
       runtime:
         - proj
-        - curl-dev
-        - tiff-dev
     description: proj dev
     test:
       pipeline:

